website/docs/d/atracker_targets.html.markdown

```diff
@@ -55,7 +55,17 @@ Nested scheme for **targets**:
 		  * Constraints: The maximum length is `1000` characters. The minimum length is `3` characters. The value must match regular expression `/^[a-zA-Z0-9 -._:\/]+$/`.
 		* `target_crn` - (String) The CRN of the LogDNA instance.
 		  * Constraints: The maximum length is `1000` characters. The minimum length is `3` characters. The value must match regular expression `/^[a-zA-Z0-9 -._:\/]+$/`.
-	* `write_status` - (List) The status of the write attempt to the target with the provided endpoint parameters.
+  * `eventstreams_endpoint` - (List) Property values for Event streams Endpoint.
+  Nested scheme for **eventstreams_endpoint**:
+    * `api_key` - (String) The IAM API key that has access to the Event streams instance.
```

Comment:
Does eventstreams_endpoint has api_key ? I see
target_crn,brokers,topic,password

Reply:
What sent to Event streams will be sent as password, but here we wanted it to be consistent and similar to the rest of our endpoints